README.md

```diff
@@ -47,6 +47,7 @@ jobs:
 | `team_access` | YAML encoded teams and their associated permissions to be granted to the created workspaces | `false` |
 | `terraform_version` | Terraform version | `1.0.3` |
 | `terraform_host` | Terraform Cloud host | `app.terraform.io` |
+| `tfe_provider_version` | Terraform Cloud provider version | `"~> 0.25.0"` |
```

Comment:
Put a big warning on this. Semver declares that 0.26 could entirely blow up this action. I can see the need to include this, namely:

* Bug is fixed by a newer version
* Bug is introduced, action continues working with older version

But this could also be pretty dangerous under normal conditions

Reply:
figured it would be nice to have some flexibility in the default, but i also see some benefit with pinning (eg we know it works!) should we switch to a pin?

Reply:
I think it's a helpful escape hatch to offer. Just include a note explaining that the action is developed against the specified version range and compatibility is not guaranteed with any custom range.

Reply:
pinned to 0.25.3, wondering if renovate would be able to help us here? Although it seems like a stretch for it to be able to update both the readme and the actions.yml file

Reply:
Also of interest here, was thinking about the risk of workspace deletion. I think it'll make sense to add an `allow_workspace_deletion` input (default false) and then check the plan for any deletions of `tfe_workspace`. Everything else can be readily re-created from config but lost state is potentially catastrophic. 

That substantially de-risks provider updates among other input changes, regardless of whether documentation was consulted.

Reply:
nice yeah good thought, that should be doable. I'll get that going in a separate pr. 

How does that relate to provider version?

Reply:
The most severe risk of a new provider version is that some attribute is `ForceNew: true` and resources are recreated. This is worse than a schema break, which will produce a straightforward error and exit early. The destroy protection guarantees that while the workflow may break, it won't go as far as throwing away persistent data unless you ask it to.